opts/capabilities.go

```diff
@@ -0,0 +1,89 @@
+package opts
+
+import (
+	"sort"
+	"strings"
+)
+
+const (
+	// AllCapabilities is a special value to add or drop all capabilities
+	AllCapabilities = "ALL"
+
+	// ResetCapabilities is a special value to reset capabilities when updating.
+	// This value should only be used when updating, not used on "create".
+	ResetCapabilities = "RESET"
```

Comment:
question: Should RESET be explicitly forbidden for service create?

Reply:
I was thinking about it, but decided to keep it simple for now

Reply:
Fine by me 👍